install_config/install/docker_registry.adoc

```diff
@@ -121,19 +123,14 @@ $ oc volume deploymentconfigs/docker-registry \
      --source='{"nfs": { "server": "<fqdn>", "path": "/path/to/export"}}'
 ----
 
-[[session-affinity-workaround]]
-There are link:#registry-known-issues[known issues] when using multiple registry
-replicas with the same NFS volume. We recommend changing the docker-registry
-service's `*sessionAffinity*` to *ClientIP*:
-
-----
-$ oc get -o yaml svc docker-registry | \
-      sed 's/\(sessionAffinity:\s*\).*/\1ClientIP/' | \
-      oc replace -f -
-----
+[NOTE]
+====
+See link:#registry-known-issues[Known Issues] if using a scaled registry with a
+shared NFS volume.
+====
```

Comment:
Yes, seems reasonable. The `sessionAffinity` defaults to `ClientIP` since 3.1 so it's ok to remove from here.